front/app/containers/SearchPage/components/AggFilterInputUpdater.tsx

```diff
@@ -60,7 +60,7 @@ abstract class AbstractAggFilterInputUpdater {
     this.configureInput();
   }
 
-  abstract onUpdateFilter(): void;
+  abstract onUpdateFilter(allowsMissingChanged?: boolean): void;
```

Comment:
this lets us hook into the change we're making to change what we push onto the site config mutation argument list.